src/machine/CoffeeMachine.java

```diff
@@ -0,0 +1,92 @@
+package machine;
+
+class CoffeeMachine {
+
+    private Supplies supplies;
+
+    // It's ok, that the type of the price is int for this task
+    private int money;
+
+    private CoffeeMachineState currentState;
+
+    CoffeeMachineState getCurrentState() {
+        return currentState;
+    }
+
+    CoffeeMachine() {
+        supplies = new Supplies(400, 540, 120, 9);
```

Comment:
It would be better to have an opportunity to pass the initial supplies and money to the constructor